cmd/ttn-lw-cli/commands/simulate.go

```diff
@@ -417,12 +429,12 @@ var (
 	simulateCommand = &cobra.Command{
 		Use:     "simulate",
 		Aliases: []string{"sim"},
-		Short:   "Simulation commands (EXPERIMENTAL)",
-		Hidden:  true,
+		Short:   "Simulation commands",
 	}
 	simulateJoinRequestCommand = &cobra.Command{
-		Use:   "join-request",
-		Short: "Simulate a join request (EXPERIMENTAL)",
+		Use:    "join-request",
```

Comment:
1. Maybe this should also have the `gateway-` prefix
2. I think the description should be more clear about the fact that this "simulates a join request from an end device, sent through a simulated gateway connection"